x-pack/plugins/enterprise_search/public/applications/shared/kibana_chrome/generate_breadcrumbs.test.ts

```diff
@@ -89,48 +93,51 @@ describe('useEuiBreadcrumbs', () => {
       },
       {
         text: 'World',
-        href: '/app/enterprise_search/world',
-        onClick: expect.any(Function),
+        // Per EUI best practices, the last breadcrumb is inactive/is not a link
       },
     ]);
   });
 
-  it('prevents default navigation and uses React Router history on click', () => {
-    const breadcrumb = useEuiBreadcrumbs([{ text: '', path: '/test' }])[0] as any;
+  describe('link behavior for non-last breadcrumbs', () => {
+    // Test helper - adds a 2nd dummy breadcrumb so that paths from the first breadcrumb are generated
+    const useEuiBreadcrumb = (breadcrumb: Breadcrumb) =>
+      useEuiBreadcrumbs([breadcrumb, { text: '' }])[0] as any;
```

Comment:
Most of the changes here are indentation changes due to the extra `describe()` block, I recommend viewing the diff with whitespace changes off.

Also I couldn't think of a better adjective than "non-last breadcrumbs", please feel free to suggest one!